frontend/src/components/TransferCreditHolder.tsx

```diff
@@ -68,6 +76,58 @@ const Wrapper = styled.div`
   height: 100%;
 `;
 
+/**
+ * Determines the corresponding transferable exam type and name for the given NEU course.
+ * @param examGroups The transferable exam groupings (either AP or IB).
+ * @param course The NEU course to query against.
+ * @returns The exam type and name as a string.
+ */
+function findExamText(
+  examGroups: TransferableExamGroup[],
+  course: ScheduleCourse
+) {
```

Comment:
let's add a return type here :)) so it will be function findExamText(...): string {}